collated/main/A0139661Y.md

```diff
@@ -0,0 +1,1790 @@
+        }
+    }
+}
+```
+###### /java/seedu/cmdo/logic/commands/StorageCommand.java
+``` java
+public class StorageCommand extends Command {
+	
+    public static final String COMMAND_WORD = "storage";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Changes the storage folder path of CMDo.\n"
+            + "Parameters: <file/path/name>"
+            + "Example: " + COMMAND_WORD
+            + " /home/Documents";
+
+    public static final String MESSAGE_SUCCESS = "cmdo.xml now saves to %1$s";
+
+    private final String filePath;
+
+    /**
+     * Created an command to change storage filepath.
+     *
+     * @throws IllegalValueException if any of the raw values are invalid
+     * 
+```
+###### /java/seedu/cmdo/logic/commands/StorageCommand.java
+``` java
+     */
+    public StorageCommand(String filePath) {
+    	this.filePath = filePath;
+    }
+
+    @Override
+    public CommandResult execute() {
+    	model.changeStorageFilePath(filePath);
+        EventsCenter.getInstance().post(new StorageFileChangedEvent(filePath));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, filePath));
+    }
+}
+```
+###### /java/seedu/cmdo/logic/LogicManager.java
+``` java
+    @Override
+    public ObservableList<ReadOnlyTask> getBlockedList() {
+    	return model.getBlockedList();
+    }
+    
+```
+###### /java/seedu/cmdo/logic/LogicManager.java
+``` java
+    @Override
+    public ObservableList<ReadOnlyTask> getAllTaskList() {
+    	return model.getAllTaskList();
+    }
+    
+    @Override
+    public ObservableList<ReadOnlyTask> getFilteredTaskList() {
+        return model.getFilteredTaskList();
+    }
+    
+```
+###### /java/seedu/cmdo/logic/LogicManager.java
+``` java
+    @Override
+    public ObservableList<ReadOnlyTask> getFilteredTaskList(boolean firstRun) {
+        return model.getFilteredTaskList(firstRun);
+    }    
+}
+```
+###### /java/seedu/cmdo/logic/parser/Blocker.java
+``` java
+	public ArrayList<LocalDateTime> checkBlocked(Task toCheck, UnmodifiableObservableList<ReadOnlyTask> blockedList)
+										throws TaskBlockedException {
+		ArrayList<LocalDateTime> dateTimeList = new ArrayList<LocalDateTime>();
+		if (toCheck.isRange()) {
+			LocalDateTime startDt = LocalDateTime.of(toCheck.getDueByDate().start, toCheck.getDueByTime().start);
+			LocalDateTime endDt = LocalDateTime.of(toCheck.getDueByDate().end, toCheck.getDueByTime().end);
+			dateTimeList.add(startDt);
+			dateTimeList.add(endDt);
+		} else {
+    		LocalDateTime dt = LocalDateTime.of(toCheck.getDueByDate().start, toCheck.getDueByTime().start);
+    		dateTimeList.add(dt);
+		}
+        if (isBlocked(dateTimeList, blockedList)) {
+        	throw new TaskBlockedException(Messages.MESSAGE_TIMESLOT_BLOCKED);
+        }
+        return dateTimeList;
+	}
+	
+	/**
+	 * Checks to see if a local date time array list of ranges falls within a blocked slot.
+	 * 
+	 * @param datesAndTimes
+	 * @return boolean based on outcome
```

Comment:
Normally, you should give a description of the @param or the @return value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value).